Sources/Crypto/BCrypt/BCryptDigest.swift

```diff
@@ -26,6 +26,10 @@ public final class BCryptDigest {
     /// - returns: BCrypt hash for the supplied plaintext data.
     public func hash(_ plaintext: LosslessDataConvertible, cost: Int = 12, salt: LosslessDataConvertible? = nil) throws -> String {
 
+        guard cost >= 4 && cost <= 31 else {
```

Comment:
Can/should we use the actual constants from the C lib here?

Reply:
👍